### DISCUSSION TITLE
Gerenciador de Interações em Redes Sociais

### DISCUSSION THREAD

**Discussion by fikani** [daniloraisi/go-ahead#3] [Ideas]
Gerenciamento de Posicionamento de Marca na internet. Abrange todas as interações por mensagens privadas, comentários, marcações, reactions e respostas em forma de imagem das redes sociais: Facebook, Instagram, Whatsapp, etc... Os engajamentos de suas contas em redes sociais.

## Problemas a resolver

- organizar as interações recebidas via rede social para que fica fácil, rápido e simples responder às mensages.
  - async crawler jobs, webhooks, etc...
  - controle de estado das mensagens recebidas: novas/respondidas/resolvidas/lixo/spam
- Identificar quais engajamentos podem gerar uma tarefa/ação de resposta para o clientes. Exemplo: reclamações, perguntas, elogios.
  - categorizaçao, gerar tarefa de resposta
- Identificar quais são os engajamentos de pessoas reais.
  - auto spam detection
- Gerar perfil e Identificar as pessoas que mais engajam com a empresa
- Analisar a reputação da empresa de acordo com as interações com as pessoas.
  - Análise de sentimento das mensagens
- Agilizar e aumentar a qualidade/quantidade das respostas às mensagens.
- Gerar respostas para as perguntas já respondidas em outros momentos, para perguntas que já estão respondidas com conteúdo institucional que ninguém lê.
  - LLM + embeddings
- Acompanhar as estatísticas de engajamento ao longo do tempo: tempo de resposta, perguntas, reclamações, conclusões de tarefas geradas através de mensagens dos clientes, análise de sentimento das mensagens recebidas.
  - insights/dashboard/analytics
  
  
  
![image](https://github.com/daniloraisi/go-ahead/assets/1130234/c801316f-0eed-4bc3-a936-3897571731d9)
![image](https://github.com/daniloraisi/go-ahead/assets/1130234/d4b9f0f7-2de0-47f6-8a8b-bdf988dab19d)

**Comment by alvez2g** [comment]
Não sei se já existe algo do tipo, mas confesso que essa ideia é incŕivel! Sem contar que durante a execução de um projeto como esse o aprendizado é muito alto, principalmente quando olhamos para a parte de análises dos dados para gerarção de estatísticas para o dashboard.

Única coisa que fico pensando pra dar segmento nisso é em relação as integrações com as APIs das redes sociais num geral, pois nem sempre oferecem os recursos necessários para algo como esse.

Mas pra mim de longe é a ideia mais interessante! hahaha
